std_msgs/Header header

uint8 temperature
uint8 information
uint8 mode
uint8 optimization
uint8 counter
uint8 heartbeat_period
uint8 facet
uint8 averaging
uint16 number_of_spots
uint16 angle_first
uint16 angle_last

float32 range_min
float32 range_max